staging/src/k8s.io/apiserver/pkg/endpoints/metrics/metrics.go

```diff
@@ -562,7 +562,7 @@ func CanonicalVerb(verb string, scope string) string {
 // LIST and APPLY from PATCH.
 func CleanVerb(verb string, request *http.Request) string {
 	reportedVerb := verb
-	if verb == "LIST" && checkIfWatch(request) {
+	if suggestedVerb := getVerbIfWatch(request); suggestedVerb == "WATCH" {
```

Comment:
This worked correctly before for metrics (because of the way it was called from in this file from verbs coming from installer).

The problem was with CleanVerb being called from:
https://github.com/kubernetes/kubernetes/blob/master/staging/src/k8s.io/apiserver/pkg/server/httplog/httplog.go#L242

Reply:
ah that's what causing the regression?